build_sdk/apiwrappers/python/openei_utilityrate_billcalculator.py

```diff
@@ -1,344 +0,0 @@
-#######################################################################
-## SDK Example: PVWatts + Utility Rate
```

Comment:
Should we keep this somewhere - SAM-documentation or SAM/samples or website LK Cookbook?

Reply:
@cpaulgilman can you please evaluate if any of these files are worth moving to a different location as reference? thanks for the suggestion @sjanzou :)

Reply:
Sorry I don't think they should be deleted. I thought build_sdk was a build folder (clearly I was wrong).